### PR TITLE
Add information about creating batch records

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @hanami/core

### CODE REVIEW

**Comment by davydovanton** [comment]
or we can add something like this:
```ruby
class BookRepository
  def create_many(list)
    command(:create, books).call(list)
  end
end
```

File: source/guides/models/repositories.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ repository = BookRepository.new
 book = repository.create(title: "Hanami")
   # => #<Book:0x007f95cbd8b7c0 @attributes={:id=>1, :title=>"Hanami", :created_at=>2016-11-13 16:02:37 UTC, :updated_at=>2016-11-13 16:02:37 UTC}>
 
+# you can create more than one record in transaction
+repository.create([{ title: "Hanami" }, { title: "Ruby"}])
```
Comment (AlfonsoUceda) [inline source/guides/models/repositories.md]: We can add too the result of this

**Comment by davydovanton** [comment]
@AlfonsoUceda I changed PR and added information how to create `#create_many` method with ROM command

@hanami/core WDYT?

File: source/guides/models/repositories.md [hunk 2]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
   # => nil
 ```
 
+## Saving more than one record
+
+For saving butch of records you need to create `#create_many` command:
```
Comment (b264) [inline source/guides/models/repositories.md]: `s/butch/a bunch/`

Comment (AlfonsoUceda) [inline source/guides/models/repositories.md]: @davydovanton can you amend this?

**Comment by AlfonsoUceda** [comment]
@davydovanton one question, do you know if the result of this query is one insert or several inserts?

File: source/guides/models/repositories.md [hunk 3]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
...
+
+```ruby
+class BookRepository
+  def create_many(list)
+    command(:create, books, result: :many).call(list)
+  end
+end
+```
+
+And after that you can use it as a `#create` method:
```
Comment (TiteiKo) [inline source/guides/models/repositories.md]: I find this sentence misleading, it may lead to people thinking the `#create` method can now create many records instead of understanding that there's a new `#create_many` method

It also can't be used as the `#create` method, as the parameters passed are really different...

I'd rephrase to `Now you can use `#create_many` as a bulk creation method:`

**Comment by davydovanton** [comment]
@AlfonsoUceda if I remember right it's only one command (`INSERT`) [source](https://github.com/rom-rb/rom-sql/blob/9ea3f06bd61a8db87ea7b71ebb9fc2bb75767f58/lib/rom/sql/relation/writing.rb#L43)

@TiteiKo great idea, I like it 👍
Thank you for fixing my bad English :D

**Comment by davydovanton** [comment]
Updated text, thank you all for review ❤️

File: source/guides/models/repositories.md [hunk 4]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
...
+
+```ruby
+class BookRepository
+  def create_many(list)
+    command(:create, books, result: :many).call(list)
+  end
+end
+```
+
+Now you can use `#create_manyas` a bulk creation method:
```
Comment (TiteiKo) [inline source/guides/models/repositories.md]: Typo here `#create_manyas`

Comment (davydovanton) [inline source/guides/models/repositories.md]: done 👍

**Comment by davydovanton** [comment]
@AlfonsoUceda @TiteiKo @b264 any other issues or I can merge this?

File: source/guides/models/repositories.md [hunk 5]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
   # => nil
 ```
 
+## Saving more than one record
+
+For saving bunch of records you need to create `#create_many` command:
```
Comment (cllns) [inline source/guides/models/repositories.md]: How about:

For saving bunch of records you need to **define a**`#create_many` command:

create #create_many looks a little strange to me :)

File: source/guides/models/repositories.md [hunk 6]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
...
+
+```ruby
+class BookRepository
+  def create_many(list)
+    command(:create, books, result: :many).call(list)
+  end
+end
+```
+
+Now you can use `#create_many` a bulk creation method:
```
Comment (cllns) [inline source/guides/models/repositories.md]: Now you can use `#create_many` **as** a bulk creation method:

File: source/guides/models/repositories.md [hunk 7]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
...
+
+```ruby
+repository = BookRepository.new
+
+repository.create_many([{ title: "Hanami" }, { title: "Ruby"}])
+# => [#<Book:0x007f95cbd8b7c0 @attributes={...}>, #<Book:0x007f95cbd8b7c0 @attributes={...}>]
```
Comment (cllns) [inline source/guides/models/repositories.md]: pedantic but I don't think the Books would have the same object id ;)

File: source/guides/models/repositories.md [hunk 8]
```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
...
+  end
+end
+```
+
+Now you can use `#create_many` a bulk creation method:
+
+```ruby
+repository = BookRepository.new
+
+repository.create_many([{ title: "Hanami" }, { title: "Ruby"}])
```
Comment (cllns) [inline source/guides/models/repositories.md]: Since it's just one command, should we do `BookRepository.new.create_many`, instead of defining a local variable?

**Comment by AlfonsoUceda** [comment]
@davydovanton can you take care of this? :P

**Comment by davydovanton** [comment]
Hey, yes. I'll do it after Japan :)

**Comment by AlfonsoUceda** [comment]
Merging because I've fixed suggestions.

**Comment by davydovanton** [comment]
thank you ❤️
